api/spec/controllers/posts.spec.js

```diff
@@ -183,4 +183,82 @@ describe("/posts", () => {
       expect(response.body.token).toEqual(undefined);
     })
   })
+  describe("PUT, when token is present", () => {
+    test("the response code is 200", async () => {
+      let post1 = new Post({message: "howdy!"});
+      await post1.save();
+      let response = await request(app)
+        .put(`/posts/${post1.id}`)
+        .set("Authorization", `Bearer ${token}`)
+        .send({token: token, message: "I meant to say Hello!"});
+        expect(response.status).toEqual(200);})
+        
```

Comment:
It might be good to fix the indentation level and formatting here — you can do this automatically using tools like Prettifier and add the extension to VS code: https://www.robinwieruch.de/how-to-use-prettier-vscode/#:~:text=Prettier%20is%20an%20opinionated%20code,control%20system%20(e.g.%20Git).

Someone reading this for the first time might think first that the test below ("the post's message gets updated") is somehow "nested" under the previous test ("the response code is 200") and that the second sort of "depends of" the first one maybe. Only when looking closer can you see that the first test is actually closed right after the `except` on line 194:
```js
expect(response.status).toEqual(200);})
```

This can cause other annoying issues when trying to deal with curly brackets and weird syntax errors — so it's always good to make sure the indentation is right. Here's an example "JS style guide" from Airbnb, which you can have a look at if you're curious: https://github.com/airbnb/javascript

Reply:
Thank you for the feedback. This is now changed. The pull request will be updated, once we have implemented the authorization of the user editing the post.